app/queues/certificate/challenge-completer-worker.server.ts

```diff
@@ -17,3 +17,5 @@ export const challengeCompleterWorker = new Worker<ChallengeCompleterData>(
   },
   { connection: redis }
 );
+
+process.on('SIGINT', () => challengeCompleterWorker.close());
```

Comment:
How is closing connections to workers based on signal interruption better than how we were doing things previously? By doing this, are we building towards the panic feature requested in #142.

Reply:
It's the same idea.  The graceful shutdown would do the same thing, but it would require us to create a connection between the server bundle and remix bundle (i.e., we'd have to get a reference to the workers in another bundle).  This way, we can do it all in the same bundle.

The SIGINT is a POSIX signal, see https://dsa.cs.tsinghua.edu.cn/oj/static/unix_signal.html#:~:text=The%20SIGINT%20signal%20is%20sent,break%22%20key%20can%20be%20used.&text=The%20SIGKILL%20signal%20is%20sent,to%20terminate%20immediately%20(kill).

Reply:
Thanks for clarifying.